x-pack/plugins/cases/server/plugin.ts

```diff
@@ -122,12 +122,18 @@ export class CasePlugin
     const router = core.http.createRouter<CasesRequestHandlerContext>();
     const telemetryUsageCounter = plugins.usageCollection?.createUsageCounter(APP_ID);
 
+    const isServerless = plugins.cloud?.isServerlessEnabled;
+
     registerRoutes({
       router,
-      routes: [...getExternalRoutes(), ...getInternalRoutes(this.userProfileService)],
+      routes: [
+        ...getExternalRoutes({ isServerless }),
+        ...getInternalRoutes(this.userProfileService),
+      ],
       logger: this.logger,
       kibanaVersion: this.kibanaVersion,
       telemetryUsageCounter,
+      isServerless,
```

Comment:
I think this is not necessary anymore

Reply:
Done in [`9319e4e` (#198378)](https://github.com/elastic/kibana/pull/198378/commits/9319e4e58f6dc272113b3dd86b446e626d3e7839)